src/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -163,6 +163,11 @@ public function getConfigTreeBuilder(): TreeBuilder
         $this->addFormatSection($rootNode, 'patch_formats', [
             'json' => ['mime_types' => ['application/merge-patch+json']],
         ]);
+        $this->addFormatSection($rootNode, 'docs_formats', [
+            'graphql' => ['mime_types' => ['application/graphql']],
+            'jsonld' => ['mime_types' => ['application/ld+json']],
+            'openapi' => ['mime_types' => ['application/vnd.openapi+json']],
```

Comment:
I think `openapi` format should be placed first (as default), then `jsonld`, and finally `html`. `graphql` format should not be available by default on docs